tests/model_fields/test_imagefield.py

```diff
@@ -469,3 +470,32 @@ def test_dimensions(self):
         # Dimensions were recalculated, and hence file should have opened.
         self.assertIs(p.mugshot.was_opened, True)
         self.assertIs(p.headshot.was_opened, True)
+
+
+@skipIf(Image is None, "Pillow is required to test ImageField")
+class NoReadTests(ImageFieldTestMixin, TestCase):
+
+    NoReadImageModel = NoReadImageModel
```

Comment:
If pillow is not installed we get an error: `NameError: name 'NoReadImageModel' is not defined`
At the top of the file there is handling for this, see...
```
# Pillow not available, create dummy classes (tests will be skipped anyway)
```

Can you do the same?

Reply:
I believe I have resolved this.